src/helper_gitlab_modify.sh

```diff
@@ -656,8 +656,21 @@ push_changes() {
 
 
 # Structure:gitlab_modify
-# source src/run_ci_job.sh && export_repo
-# Write function that exportis the test-repository to a separate external folder.
+#######################################
+# Checks if the target folder already exist and then deletes it.  
```

Comment:
This method was poorly written by me. We could improve it by ensuring the global variable is changed to a local variable, such that the function is called with specificity.

To prevent having to refactor the current code-base outside this file, it can be resolved by including a TODO.